darkmoon/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+</body>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+
+    </div>
+
+    <h1>
+
+        <label for="username" class="opSystem" data-icon="u"></label>
+
+        <input id="username" name="opSystem" required="required" type="text" placeholder="Operating System" />
+
+    </h1>
+
+    <h1>
+
+        <label for="password" class="vers" data-icon="p"> </label>
+
+        <input id="password" name="password" required="required" type="text" placeholder="Version" />
+
+    </h1>
+    <h1>
+
+        <label for="username" class="bu" data-icon="u"> </label>
+
+        <input id="bu" name="username" required="required" type="text" placeholder="Build" />
+    </h1>
+
+    <form action="/action_page.php">
```

Comment:
Please remove references to the PHP targets and point these actions to their respective endpoints within the Darkmoon API.